### ISSUE TITLE
Feature request for BAM tag on STARSolo

### ISSUE THREAD

**Issue by cnk113** [alexdobin/STAR#1560]
Hey Alex,

I was wondering if it was possible to get a similar feature from CellRanger which is to tag (RE:) if the reads are intronic (N), exonic (E), or intergenic (I)? I'm assuming this is pulled from the GTF file.

Best,
Chang

**Comment by alexdobin** [comment]
Hi Chang,

I can output Exonic/Intronic tag for ```--soloFeatures GeneFull_ExonOverIntron and GeneFull_Ex50pAS```.
There is no "intergenic" assessment, the reads that are not intronic or exonic, could be either intergenic or belong to novel transcripts.

Cheers
Alex

**Comment by cnk113** [comment]
Yeah you're probably right on the intergenic assignment, I'm not too sure how they make that classification, but it's likely the remaining category after assigning exonic/intronic.

**Comment by hai178912522** [comment]
Is this feature ready? Can 2.7.11a distinguish between exonic, intronic, etc. in bam files?

**Comment by Hoohm** [comment]
The TAG does not exit atm, I tried running with RE as outSAMattributes but it doesn't recognize it.
